Formula/go.rb

```diff
@@ -3,6 +3,10 @@ class Go < Formula
   homepage "https://golang.org"
 
   stable do
+    # Version bumps of this formula require formulae that depend on Go at
```

Comment:
In the case of godep, it's currently required every time even on revision bumps. The other revision bumps in this PR are probably unnecessary altogether.